README.md

```diff
@@ -0,0 +1,265 @@
+If you want to specificy your own browser list, you can do that via the `browsers` argument (default is: `["chrome", "edge", "internet explorer", "firefox", "safari", "opera"]`).
+
+```py
+from fake_useragent import UserAgent
+ua = UserAgent(browsers=['edge', 'chrome'])
+ua.random
+```
+
+`fake-useragent` store collected data at your os temp dir, like `/tmp`
+
+If you want to update saved database just:
+
+```py
+from fake_useragent import UserAgent
+ua = UserAgent()
+ua.update()
+```
+
+If you don't want cache database or no writable file system:
+
+```py
+from fake_useragent import UserAgent
+ua = UserAgent(cache=False)
+```
+
+Sometimes, [useragentstring.com](http://useragentstring.com) changes their html or is down, in such case `fake-useragent` uses the following [hosted cache server](https://useragent.melroy.org/cache.json) as fallback.
+
+If you don't want to use hosted cache server (version 0.1.5 added).
+
+```py
+from fake_useragent import UserAgent
+ua = UserAgent(use_cache_server=False)
+```
+
+In very rare case, if hosted cache server and sources will be
+unavailable `fake-useragent` wont be able to download data: (version 0.1.3 added)
+
+```py
+from fake_useragent import UserAgent
+ua = UserAgent()
+
+# Traceback (most recent call last):
+#   ...
+# fake_useragent.errors.FakeUserAgentError
+
+# you can catch it via
+
+from fake_useragent import FakeUserAgentError
+
+try:
+    ua = UserAgent()
+except FakeUserAgentError:
+    pass
+```
+
+If you will try to get unknown browser: (version 0.1.3 changed)
+
+```py
+from fake_useragent import UserAgent
+ua = UserAgent()
+ua.best_browser
+# Traceback (most recent call last):
+#   ...
+# fake_useragent.errors.FakeUserAgentError
+```
+
+You can completely disable ANY annoying exception with adding `fallback`: (version 0.1.4 added)
+
+```py
+import fake_useragent
+
+ua = fake_useragent.UserAgent(fallback='your favorite Browser')
+# in case if something went wrong, one more time it is REALLY!!! rare case
+ua.random == 'your favorite Browser'
+```
+
+Want to control location of data file? (version 0.1.4 added)
+
+```py
+import fake_useragent
+
+# I am STRONGLY!!! recommend to use version suffix
+location = '/home/user/fake_useragent%s.json' % fake_useragent.VERSION
+
+ua = fake_useragent.UserAgent(path=location)
+ua.random
+```
+
+If you need to safe some attributes from overriding them in UserAgent by `__getattr__` method
+use `safe_attrs` you can pass there attributes names.
+At least this will prevent you from raising FakeUserAgentError when attribute not found.
+
+For example, when using fake*useragent with `injections <https://github.com/tailhook/injections>`* you need to:
+
+```py
+import fake_useragent
+
+ua = fake_useragent.UserAgent(safe_attrs=('__injections__',))
+```
+
+Please, do not use if you don't understand why you need this.
+This is magic for rarely extreme case.
+
+### Experiencing issues???
+
+Make sure that you using latest version!!!
+
+```sh
+pip install -U fake-useragent
```

Comment:
`pip install` has [options](https://pip.pypa.io/en/stable/cli/pip_install/#options) for user, update, update-strategy, and use-pep517 so it is better to be explicit.
```suggestion
pip install --update fake-useragent
```

Reply:
You have only `-U` or `--upgrade`: https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-U ??

Reply:
Thx!  Fixed inline.